recipes/bwa/meta.yaml

```diff
@@ -1,29 +1,34 @@
+{% set version = "0.7.16" %}
+
 package:
   name: bwa
-  version: "0.7.15"
+  version: {{ version }}
 
 build:
-  number: 1
+  number: 0
 
 source:
-  fn: v0.7.15.tar.gz
-  url: https://github.com/lh3/bwa/archive/v0.7.15.tar.gz
-  md5: 54fdee953c5c256d36885a1c5c6b118c
+  fn: v{{ version }}.tar.gz
+  url: https://github.com/lh3/bwa/archive/v{{ version }}.tar.gz
+  sha256: a9187f8999a5624653cb9f317b162d0fa6bd0b85aceeefff66950aed5d31e1dc
 
 requirements:
   build:
     - gcc   # [not osx]
     - llvm  # [osx]
-    - zlib
+    - zlib {{CONDA_ZLIB}}*
   run:
-    - zlib
+    - zlib {{CONDA_ZLIB}}*
     - libgcc  # [not osx]
+    - perl
 
 test:
   commands:
     - bwa 2>&1 | grep 'index sequences in the'
```

Comment:
@johanneskoester - this is causing mulled to fail (the `'` causes this command to break out of the shell escaping) - you need to use `"index sequences in the"` instead)

Reply:
Thanks!